spra-play-server/src/test/scala/controllers/AdminControllerSpec.scala

```diff
@@ -802,23 +803,25 @@ class AdminControllerSpec extends PlayPostgresSpec with AdminUtils {
 
     "don't fail if we send a null in an optional parameter" in withApiClient { client =>
       val name = "wiringbits"
+      val lastName = "test"
       val email = "test@wiringbits.net"
       val password = "wiringbits"
-      val request = AdminCreateTable.Request(Map("name" -> name, "email" -> email, "password" -> password))
+      val request =
+        AdminCreateTable.Request(Map("name" -> name, "last_name" -> lastName, "email" -> email, "password" -> password))
       client.createItem("users", request).futureValue
       val responseMetadata1 =
         client.getTableMetadata(usersSettings.tableName, List("name", "ASC"), List(0, 9), "{}").futureValue
       val id = responseMetadata1.head("id")
 
-      val json = """{"email":null}"""
+      val json = """{"last_name":"null"}"""
```

Comment:
Please create a ticket, we need to find a way to deal with proper null values instead of the string "null"